TAP version 14
# Subtest: GET /api/analytics/1rm-progression - T017
    not ok 1 - Method 'GET' already declared for route '/api/analytics/volume-trends'
      ---
      stack: |
        Object.addNewRoute (node_modules/fastify/lib/route.js:371:19)
        Object.route (node_modules/fastify/lib/route.js:265:19)
        Object.prepareRoute (node_modules/fastify/lib/route.js:167:18)
        Object._get [as get] (node_modules/fastify/fastify.js:262:34)
        analyticsRoutes (src/routes/analytics.ts:309:11)
        Plugin.exec (node_modules/avvio/lib/plugin.js:125:28)
        Boot._loadPlugin (node_modules/avvio/boot.js:432:10)
      at:
        fileName: node_modules/fastify/lib/route.js
        lineNumber: 371
        columnNumber: 19
        typeName: Object
        methodName: addNewRoute
        functionName: Object.addNewRoute
      type: FastifyError
      code: FST_ERR_DUPLICATED_ROUTE
      statusCode: 500
      tapCaught: returnedPromiseRejection
      source: >2
                  const isDuplicatedRoute = error.message.includes(`Method '${opts.method}' already declared for route`)
                  if (isDuplicatedRoute) {
                    throw new FST_ERR_DUPLICATED_ROUTE(opts.method, opts.url)
        ------------------^
                  }
      ...
    
    1..1
not ok 1 - GET /api/analytics/1rm-progression - T017 # time=72.979ms
  ---
  at:
    fileName: tests/contract/analytics.test.ts
    lineNumber: 12
    columnNumber: 5
    isToplevel: true
  source: |2
     */
  
    tap.test('GET /api/analytics/1rm-progression - T017', async (t) => {
    ----^
      const app = await buildApp();
  ...

# Subtest: GET /api/analytics/volume-trends - T018
    not ok 1 - Method 'GET' already declared for route '/api/analytics/volume-trends'
      ---
      stack: |
        Object.addNewRoute (node_modules/fastify/lib/route.js:371:19)
        Object.route (node_modules/fastify/lib/route.js:265:19)
        Object.prepareRoute (node_modules/fastify/lib/route.js:167:18)
        Object._get [as get] (node_modules/fastify/fastify.js:262:34)
        analyticsRoutes (src/routes/analytics.ts:309:11)
        Plugin.exec (node_modules/avvio/lib/plugin.js:125:28)
        Boot._loadPlugin (node_modules/avvio/boot.js:432:10)
      at:
        fileName: node_modules/fastify/lib/route.js
        lineNumber: 371
        columnNumber: 19
        typeName: Object
        methodName: addNewRoute
        functionName: Object.addNewRoute
      type: FastifyError
      code: FST_ERR_DUPLICATED_ROUTE
      statusCode: 500
      tapCaught: returnedPromiseRejection
      source: >2
                  const isDuplicatedRoute = error.message.includes(`Method '${opts.method}' already declared for route`)
                  if (isDuplicatedRoute) {
                    throw new FST_ERR_DUPLICATED_ROUTE(opts.method, opts.url)
        ------------------^
                  }
      ...
    
    1..1
not ok 2 - GET /api/analytics/volume-trends - T018 # time=18.981ms
  ---
  at:
    fileName: tests/contract/analytics.test.ts
    lineNumber: 107
    columnNumber: 5
    isToplevel: true
  source: |
    });
  
    tap.test('GET /api/analytics/volume-trends - T018', async (t) => {
    ----^
      const app = await buildApp();
  ...

# Subtest: GET /api/analytics/consistency - T019
    not ok 1 - Method 'GET' already declared for route '/api/analytics/volume-trends'
      ---
      stack: |
        Object.addNewRoute (node_modules/fastify/lib/route.js:371:19)
        Object.route (node_modules/fastify/lib/route.js:265:19)
        Object.prepareRoute (node_modules/fastify/lib/route.js:167:18)
        Object._get [as get] (node_modules/fastify/fastify.js:262:34)
        analyticsRoutes (src/routes/analytics.ts:309:11)
        Plugin.exec (node_modules/avvio/lib/plugin.js:125:28)
        Boot._loadPlugin (node_modules/avvio/boot.js:432:10)
      at:
        fileName: node_modules/fastify/lib/route.js
        lineNumber: 371
        columnNumber: 19
        typeName: Object
        methodName: addNewRoute
        functionName: Object.addNewRoute
      type: FastifyError
      code: FST_ERR_DUPLICATED_ROUTE
      statusCode: 500
      tapCaught: returnedPromiseRejection
      source: >2
                  const isDuplicatedRoute = error.message.includes(`Method '${opts.method}' already declared for route`)
                  if (isDuplicatedRoute) {
                    throw new FST_ERR_DUPLICATED_ROUTE(opts.method, opts.url)
        ------------------^
                  }
      ...
    
    1..1
not ok 3 - GET /api/analytics/consistency - T019 # time=16.417ms
  ---
  at:
    fileName: tests/contract/analytics.test.ts
    lineNumber: 208
    columnNumber: 5
    isToplevel: true
  source: |
    });
  
    tap.test('GET /api/analytics/consistency - T019', async (t) => {
    ----^
      const app = await buildApp();
  ...

1..3
Database connection closed
